boxoffice/models/item_collection.py

```diff
@@ -19,3 +20,22 @@ class ItemCollection(BaseScopedNameMixin, db.Model):
     organization = db.relationship(Organization, backref=db.backref('item_collections', cascade='all, delete-orphan'))
     parent = db.synonym('organization')
     tax_type = db.Column(db.Unicode(80), nullable=True, default=u'GST')
+
+    @property
```

Comment:
This is better off as a function, since that's the convention we're using. Also, this function will take a parameter. Let's keep this definition consistent with how we're using it elsewhere?

Reply:
Fixed